app-server/app/src/main/scala/scene/Scene.scala

```diff
@@ -1,27 +1,82 @@
 package com.azavea.rf.scene
 
 import java.sql.Timestamp
+import java.util.UUID
 
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.{Failure, Success, Try}
 
+import com.lonelyplanet.akka.http.extensions.{PageRequest, Order}
+import slick.lifted._
+
+import com.azavea.rf.AkkaSystem
 import com.azavea.rf.datamodel.latest.schema.tables._
-import com.azavea.rf.utils.Database
+import com.azavea.rf.datamodel.enums._
+import com.azavea.rf.utils.{Database => DB, PaginatedResponse}
+import com.azavea.rf.datamodel.driver.ExtendedPostgresDriver
+
+
+case class CreateScene(
+  organizationId: UUID,
+  ingestSizeBytes: Int,
+  visibility: Visibility,
+  resolutionMeters: Float,
+  tags: List[String],
+  datasource: String,
+  sceneMetadata: Map[String, Any],
+  cloudCover: Option[Float],
+  acquisitionDate: Option[java.sql.Timestamp],
+  thumbnailStatus: JobStatus,
+  boundaryStatus: JobStatus,
+  status: JobStatus,
+  sunAzimuth: Option[Float],
+  sunElevation: Option[Float]
+) {
+  def toScene(userId: String): ScenesRow = {
+    val now = new Timestamp((new java.util.Date()).getTime())
+    ScenesRow(
+      UUID.randomUUID, // primary key
+      now, // createdAt
```

Comment:
These annotations look helpful. Maybe the bucket case class should get the same treatment.